wagtail/test/search/models.py

```diff
@@ -135,3 +135,20 @@ class UnindexedBook(index.Indexed, models.Model):
     tags = TaggableManager()
 
     search_fields = []
+
+
+class EventPage(index.Indexed, models.Model):
```

Comment:
This needs generating migrations.
The `Book` models has a `publication_date` field, why not add a nullable `publication_time` field on it, then generate the migration?